dace/codegen/targets/cpu.py

```diff
@@ -1452,7 +1452,7 @@ def _generate_NestedSDFG(
             nested_stream.write(local_code)
 
             # Process outgoing memlets with the internal SDFG
-            self.process_out_memlets(sdfg,
+            codegen.process_out_memlets(sdfg,
                                      state_id,
```

Comment:
yapf!

Reply:
But that is already yapfed :/
Is there some yapf style guide?

Reply:
`state_id` and the following arguments are not indented properly, so it seems not. Are you running yapf 0.30 or newer? It should use the `.style.yapf` in the DaCe top-level directory.